### PR TITLE
updating poetry

### PR DESCRIPTION
**Pull request by DJDMVP**
Updating and removing some dependencies

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,15 +4,15 @@ version = "0.1.0"
...
 license = "MIT"
+#package.mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10"
 pip = "^22.0.4"
 mavsdk = "*"
 numpy = "^1.21.2"
-opencv-python = "^4.5.3.56"
+opencv-python = "^4.8.0."
```
Comment (cjhr95) [inline pyproject.toml]: Harper listed the version as 4.8.0.76 in the Discord, is this for future-proofing the poetry dependency?

File: pyproject.toml [hunk 2]
```diff
@@ -4,15 +4,15 @@ version = "0.1.0"
 description = ""
 authors = []
 license = "MIT"
+#package.mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10"
```
Comment (cjhr95) [inline pyproject.toml:10]: Has torch been updated to support Python 3.12 yet?

Comment (DJDMVP) [reply pyproject.toml:10]: Simulation says they would be held back cause of airsim hasn't been upgraded to 3.10, but pytorch has at least partial if not mostly/full implementation on 3.12

Comment (cjhr95) [reply pyproject.toml:10]: Alright, so we don't need to update Python until AirSim supports 3.10 then.

**Review by cjhr95 — APPROVED** [review]

**Pull request merged**